common/Cargo.toml

```diff
@@ -20,6 +20,8 @@ raw-cpuid = "10.2.0"
 rand = { version = "0.8.4", default-features = false }
 rand_hc = "0.3.1"
 uart_16550 = "0.2.18"
+serde-json-core = "0.5.0"
```

Comment:
Something like postcard is likely lighter while still supporting serde and no_std.

Reply:
I will change JSON to postcard, in a future commit. Thanks!